server/apps/immich/src/api-v1/asset/asset.service.ts

```diff
@@ -166,6 +194,10 @@ export class AssetService {
         res.set({
           'Content-Type': 'image/jpeg',
         });
+        if (!asset.resizePath) {
+          Logger.error('Error serving IMAGE asset for web ');
```

Comment:
I based those lines in the ones below in line 198-199:

```typescript
Logger.error('Error serving IMAGE asset ', e);
throw new InternalServerErrorException(`Failed to serve image asset ${e}`, 'ServeFile');
```
Looks like all `InternalServerErrorException` in this file use `ServeFile` as the second param. So I think it is consistent with the other ones.

Agree, we could also add `ServeFile` as the second argument of `Logger.error`, since we don't have an error to pass as in there (like in the other places).

Let me know what you think